test/format/ops.c

```diff
@@ -1130,6 +1130,8 @@ ops(void *arg)
         if (table->type != ROW && !table->mirror)
             WT_ORDERED_READ(max_rows, table->rows_current);
         tinfo->keyno = mmrand(&tinfo->data_rnd, 1, (u_int)max_rows);
+        if (TV(OPS_PARETO))
+            tinfo->keyno = testutil_pareto(tinfo->keyno, 1, (u_int)max_rows, 20);
```

Comment:
Not a fan of the hardcoded 20 but don't have a read on what value would be good there, @ddanderson.